connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -197,8 +208,12 @@ ble_error_t GattServer::insert_characteristic(
     attsAttr_t *&attribute_it
 )
 {
+    tr_info("Insert characteristic %s",
+            trace_uuid(characteristic->getValueAttribute().getUUID()));
```

Comment:
The `trace_uuid` should be replaced into `to_string`.

Reply:
Indeed. Resolved in commit 8f28da36be2c0da2b2abfb5ba2120a509d7bb5cc.